orange/src/test/resources/purple/variants.vcf

```diff
@@ -0,0 +1,32 @@
+##fileformat=VCFv4.2
+##FILTER=<ID=PASS,Description="All filters passed">
+##FILTER=<ID=PON,Description="Filter PON variant">
+##FILTER=<ID=PONArtefact,Description="Filter PON artefact">
+##FILTER=<ID=dedupIndel,Description="Variant duplicate SNV/MNV vs INDEL">
+##FILTER=<ID=dedupMatch,Description="Variant duplicate with different read contexts">
+##FILTER=<ID=dedupMixedGermlineSomatic,Description="Variant duplicate mixed somatic/germline">
+##FILTER=<ID=dedupMnv,Description="Filter duplicate MNV">
+##FILTER=<ID=dedupSnvMnv,Description="Variant duplicate MNV vs SNV">
+##FILTER=<ID=maxGermlineAltSupport,Description="Excess germline alt support">
+##FILTER=<ID=maxGermlineRelRawBaseQual,Description="Excess germline relative quality">
+##FILTER=<ID=maxGermlineVAF,Description="Excess germline VAF">
+##FILTER=<ID=minAvgBaseQual,Description="Variant average base quality below limit">
+##FILTER=<ID=minGermlineDepth,Description="Insufficient germline depth">
+##FILTER=<ID=minTumorQual,Description="Insufficient tumor quality">
+##FILTER=<ID=minTumorVAF,Description="Insufficient tumor VAF">
+##FILTER=<ID=strandBias,Description="Variant exceeds strand bias limit">
+##FORMAT=<ID=ABQ,Number=1,Type=Integer,Description="Average calculated base quality">
+##FORMAT=<ID=AD,Number=R,Type=Integer,Description="Allelic depths for the ref and alt alleles in the order listed">
+##FORMAT=<ID=AF,Number=1,Type=Float,Description="Allelic frequency calculated from read context counts as (Full + Partial + Core + Realigned + Alt) / Coverage">
+##FORMAT=<ID=DP,Number=1,Type=Integer,Description="Approximate read depth (reads with MQ=255 or with bad mates are filtered)">
+##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
+##FORMAT=<ID=RABQ,Number=2,Type=Integer,Description="Raw allelic base quality">
+##FORMAT=<ID=RAD,Number=2,Type=Integer,Description="Raw allelic depth">
+##FORMAT=<ID=RC_CNT,Number=7,Type=Integer,Description="Read context counts [Full, Partial, Core, Realigned, Alt, Reference, Total]">
+##FORMAT=<ID=RC_IPC,Number=1,Type=Integer,Description="Read context improper pair count">
+##FORMAT=<ID=RC_JIT,Number=3,Type=Integer,Description="Read context jitter [Shortened, Lengthened, QualityPenalty]">
+##FORMAT=<ID=RC_QUAL,Number=7,Type=Integer,Description="Read context quality [Full, Partial, Core, Realigned, Alt, Reference, Total]">
+##FORMAT=<ID=RDP,Number=1,Type=Integer,Description="Raw read depth">
+##FORMAT=<ID=SB,Number=1,Type=Float,Description="Strand bias - percentage of first-in-pair reads">
+#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	COLO829v003R	COLO829v003T
```

Comment:
Minor detail but I generally use proper test names rather than "COLO829v003R". In `test_run` I use `ref_sample` and `tumor_sample` for example.